script.js

```diff
@@ -1,4 +1,113 @@
+//1. Player 1 to roll dice
+//2. Player 1 to decide order of dice
+//3. Player 2 to roll dice
+//4. Player 2 to decide order of dice
+//5. Determine if Player 1 or 2 wins
+//6. Game continues with running score being tabulated
+
+var diceRoll_1 = "";
+var diceRoll_2 = "";
+var diceResult = "";
+var player = "Player 1";
+var winner = "";
+var gameMode = "start";
+var myOutputValue = "";
+var player1score = [];
+var player2score = [];
+var player1runningscore = "";
+var player2runningscore = "";
+var leaderboard = "";
+
+var diceRoll = function (input) {
+  if (input == "roll") {
+    diceRoll_1 = Math.floor(Math.random() * 6) + 1;
+    diceRoll_2 = Math.floor(Math.random() * 6) + 1;
+  }
+};
+
+var diceCombined = function (input) {
+  if (input == 1) {
+    diceResult = parseInt(diceRoll_1.toString() + diceRoll_2.toString());
```

Comment:
diceRoll1 & diceRoll_2 should already be an integer, so type conversion is not necessary here!